ql/src/java/org/apache/hadoop/hive/ql/metadata/HiveStorageHandler.java

```diff
@@ -257,6 +258,16 @@ default boolean supportsPartitionTransform() {
     return false;
   }
 
+  /**
+   * Return the partition transform specification table property key together with the specification value in json
```

Comment:
Maybe mention in the docs that this should be overridden only if the `supportsPartitionTransform()` method is true?

Reply:
Good point will add that.